cms/src/routes/genomicVariants.js

```diff
@@ -44,9 +44,9 @@ variantsRouter.get('/clear/:name', async (req, res) => {
     if (!result) {
       res.status(400).json({ success: false, message: 'Model does not exist.' });
     }
-    res.status(200).json({ success: true, model });
+    res.status(200).json({ success: true, model: name });
   } catch (error) {
-    logger.error({ ...error, model: name }, 'Unexpected error clearing genomic-variants for model');
+    logger.error(...error, `Unexpected error clearing genomic-variants for model ${name}`);
```

Comment:
dont need the spread operator before error since its not wrapped in another object.

just `logger.error(error, 'message')` works best